hw_diag/utilities/hardware.py

```diff
@@ -0,0 +1,118 @@
+import logging
+import os
+from time import sleep
+
+from hw_diag.utilities.shell import config_search_param
+
+
+def set_diagnostics_bt_lte(diagnostics):
+    devices = {
+        'BT': '0a12',
+        'LTE': '2c7c'
+    }
+
+    for dev_type, dev_addr in devices.items():
+        resp = os.popen('grep 0a12 /sys/bus/usb/devices/*/idVendor').read()
```

Comment:
shouldn't this have a variable instead of just `0a12` after the `grep`?

As currently it won't find the LTE will it?

Reply:
Thanks @shawaj nice catch I'll get this sorted.